src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -40,130 +41,118 @@
  * These methods are mapped to API endpoints by Spark.
  */
 public class FeedSourceController {
-    public static final Logger LOG = LoggerFactory.getLogger(FeedSourceController.class);
-    public static JsonManager<FeedSource> json =
-            new JsonManager<>(FeedSource.class, JsonViews.UserInterface.class);
+    private static final Logger LOG = LoggerFactory.getLogger(FeedSourceController.class);
+    private static JsonManager<FeedSource> json = new JsonManager<>(FeedSource.class, JsonViews.UserInterface.class);
     private static ObjectMapper mapper = new ObjectMapper();
 
-    public static FeedSource getFeedSource(Request req, Response res) {
+    /**
+     * Spark HTTP endpoint to get a single feed source by ID.
+     */
+    private static FeedSource getFeedSource(Request req, Response res) {
         return requestFeedSourceById(req, Actions.VIEW);
     }
 
-    public static Collection<FeedSource> getAllFeedSources(Request req, Response res) {
+    /**
+     * Spark HTTP endpoint that handles getting all feed sources for a handful of use cases:
+     * - for a single project (if projectId query param provided)
+     * - for the entire application
+     */
+    private static Collection<FeedSource> getAllFeedSources(Request req, Response res) {
         Collection<FeedSource> feedSourcesToReturn = new ArrayList<>();
         Auth0UserProfile requestingUser = req.attribute("user");
         String projectId = req.queryParams("projectId");
-        Boolean publicFilter = req.pathInfo().contains("public");
-        String userId = req.queryParams("userId");
-
-        if (projectId != null) {
-            for (FeedSource source: Persistence.feedSources.getAll()) {
-                String orgId = source.organizationId();
-                if (
-                    source != null && source.projectId != null && source.projectId.equals(projectId)
-                    && requestingUser != null && (requestingUser.canManageFeed(orgId, source.projectId, source.id) || requestingUser.canViewFeed(orgId, source.projectId, source.id))
-                ) {
-                    // if requesting public sources and source is not public; skip source
-                    if (publicFilter && !source.isPublic)
-                        continue;
-                    feedSourcesToReturn.add(source);
-                }
-            }
-        } else if (userId != null) {
-            // request feed sources a specified user has permissions for
-            Auth0UserProfile user = getUserById(userId);
-            if (user == null) return feedSourcesToReturn;
-
-            for (FeedSource source: Persistence.feedSources.getAll()) {
+        // Determine whether request is coming from a public HTTP endpoint.
+        boolean publicFilter = req.pathInfo().contains("public");
+        List<FeedSource> allFeedSources = Persistence.feedSources.getAll();
```

Comment:
I'm guessing we're not going to have that many feedSources to search through so it's fine to keep this for now, but it seems like there could be a way to query the Persistence to get the desired feed sources.

Reply:
Indeed, good point. But per your other good point, we might just need to remove this because most traces of the public API have been ripped out already.